CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# 1.2.20
+
+## Enhancements
+
+* Improved error message when a column has a incorrect type
+
+## Fixes
+
+* filter pushdown in cases like `p in (NULL, 1, 2)`
+* glitch with duplicate root parentExternalId in Asset hierarchy builder
```

Comment:
Follow the style of the other comments, Capitalized first word and end the line with a dot.

Rather than "glitch with", do we mean "Duplicated root parentExternalId values are now allowed in `assethierarchy`."?